apps/meteor/client/views/admin/subscription/components/FeatureUsageCard.tsx

```diff
@@ -13,17 +13,26 @@ export type CardProps = {
 	title: string;
 	infoText?: ReactNode;
 	upgradeButton?: ReactNode;
+	isHorizontallyCenteredOnly?: boolean;
 };
 
 const FeatureUsageCard = ({ children, card }: FeatureUsageCardProps): ReactElement => {
-	const { title, infoText, upgradeButton } = card;
+	const { title, infoText, upgradeButton, isHorizontallyCenteredOnly } = card;
+
 	return (
 		<Card height='full'>
 			<CardTitle>
 				{title} {infoText && <InfoTextIconModal title={title} infoText={infoText} />}
 			</CardTitle>
 			<CardBody>
-				<Box h='full' w='full'>
+				<Box
+					h='full'
+					w='full'
+					display='flex'
+					alignItems='center'
+					justifyContent={isHorizontallyCenteredOnly ? 'flex-start' : 'center'}
```

Comment:
instead of passing this `isHorizontallyCenteredOnly` as a prop, couldn't you do this conditional styling directly on the parent, on the `children` wrapper?

Reply:
If I understood your suggestion, this would require raising the box that is currently inside of the FeatureUsageCard to each card and then changing the justifyContent according to which cards need to be centered. I think this would require a lot of unnecessary duplication of code as all of the styling of this box besides the justifyContent is default to all cards. What I could do instead is change the boolean to a string called justifyContent and just pass it to the justifyContent of the children's parent box. Wdyt? If I didn't get the suggestion right please let me know.

Reply:
So maybe create a component:
`FeatureUsageCardBody` that receives the styling props, and this component would be passed as children of the `FeatureUsageCard`
eg
```
		<FeatureUsageCard card={card}>
			<FeatureUsageCardBody justifyContent='flex-start'>
					(...........)
			</FeatureUsageCardBody justifyContent='flex-start'>
		</FeatureUsageCard>
```
I just don't like styling props being inside a `card` object. This prop should only receive values that are relative to the card content (title, text, etc).